roles/openshift_cli/defaults/main.yml

```diff
@@ -1,2 +1,2 @@
 ---
-openshift_version: "{{ openshift_image_tag | default(openshift.common.image_tag) | default('') }}"
+openshift_version: "{{ openshift_image_tag | default(openshift.docker.openshift_image_tag) | default('') }}"
```

Comment:
similar thing here with the default filters needing to be nested rather than chained.